search-commons/src/test/java/no/unit/nva/search2/OpensearchAwsTest.java

```diff
@@ -0,0 +1,568 @@
+package no.unit.nva.search2;
```

Comment:
Look into this and see how and where it is used

Reply:
I will remove it from this PR, and add a new task for this test....